python/pickle/PyROOT_writetests.py

```diff
@@ -75,7 +75,7 @@ def test4WriteROOTObjInNamespace( self ):
    def test5WriteCustomTypes( self ):
       """Test writing PyROOT custom types"""
 
-      o = [ ROOT.Long(123), ROOT.Double(123.123) ]
+      o = [ctypes.c_long(123), ctypes.c_double(123.123)]
```

Comment:
For example, `ROOT.Long` and `ROOT.Double` still need to be tested in the old PyROOT!